File: rock, paper, scissors/rock.js
let choices = ['rock', 'paper', 'scissors']
let player = document.querySelector('#player')
let computer = document.querySelector('#computer')
let display = document.querySelector('#result')
let rock = document.querySelector('#rock')
let paper = document.querySelector('#paper')
let scissors = document.querySelector('#scissors')
let playerScoreDisplay = document.querySelector('.playerScoreDisplay')
let compScoreDisplay = document.querySelector('.compScoreDisplay')
let playerScore = 0
let compScore = 0

function playGame(playerChoice){
    let compChoice = choices[Math.floor(Math.random()*3)]
    let result = ""

    if(playerChoice === compChoice){
        result = "IT' A TIE!" 
    }
    else{
        switch(playerChoice){
            case "rock":
            result = (compChoice === "scissors") ? "YOU WIN🎉" : "YOU LOSE😭";
            break;
            case "paper":
            result = (compChoice === "rock") ? "YOU WIN🎉" : "YOU LOSE😭";
            break;
            case "scissors":
            result = (compChoice === "paper") ? "YOU WIN🎉" : "YOU LOSE😭";
            break;
        }
    }

    
   player.textContent = `PLAYER: ${playerChoice}`
   computer.textContent = `COMPUTER: ${compChoice}`
   display.textContent = result

   display.classList.remove("green", "red")
   switch(result){
    case "YOU WIN🎉":
        display.classList.add("green");
        playerScore++;
        playerScoreDisplay.textContent = playerScore;
        break;
        case "YOU LOSE😭":
            display.classList.add("red");
            compScore++;
            compScoreDisplay.textContent = compScore;
            break;
   }
}


/*
rock.addEventListener('click',function(){
    alert('Your choice is rock')
}
)
paper.addEventListener('click',function(){
    alert('Your choice is paper')
}
)
scissors.addEventListener('click',function(){
    alert('Your choice is scissors')
}
)*/

File: Project power/python/rock.py
import customtkinter as ctk
import random

ctk.set_appearance_mode("dark")
ctk.set_default_color_theme("blue")

app = ctk.CTk()
app.title("ROCK-PAPER-SCISSORS")
app.geometry("400x350")

result_label = ctk.CTkLabel(app, text="", font=("Arial", 18))
result_label.pack(pady=20)

def playGame(player_choice):
    choices = ["rock", "paper", "scissors"]
    comp_choice = random.choice(choices)

    if player_choice == comp_choice:
        result = f"Computer chose {comp_choice}. It's a tie!"
    elif (player_choice == "rock" and comp_choice == "scissors") or \
         (player_choice == "paper" and comp_choice == "rock") or \
         (player_choice == "scissors" and comp_choice == "paper"):
        result = f"Computer chose {comp_choice}. You win!"
    else:
        result = f"Computer chose {comp_choice}. You lose!"
        
        # result_label.configure(text=f"Computer chose {comp_choice}. You lose!", font=ctk.CTkFont(size=14), text_color="green")

    result_label.configure(text=result)

rock_button = ctk.CTkButton(app, text="ROCK 👊🏾", command=lambda: playGame("rock"))
rock_button.pack(pady=10)

paper_button = ctk.CTkButton(app, text="PAPER 🖐", command=lambda: playGame("paper"))
paper_button.pack(pady=10)

scissors_button = ctk.CTkButton(app, text="SCISSORS ✂", command=lambda: playGame("scissors"))
scissors_button.pack(pady=10)


app.mainloop()
